### ISSUE TITLE
Tabris CLI - How to target specific emulator device?

### ISSUE THREAD

**Issue by bev-on** [eclipsesource/tabris-js#1490]
In Cordova, we can run application on specific device using ```cordova run ios --target="iPad-Air, 9,2"``` for example. Also we can get list of available devices using ```--list``` option.

Problem is that although tabris-cli documentation mentions that we can use Cordova options ```tabris run [options] <platform> [cordova-platform-opts]```, I wasn't able to target any specific device.

```
$ tabris run ios --target="iPad-Air, 9.2"

   error: unknown option `--target'
```

Same ```error: unknown option``` problem with --list option.

If I run only ```tabris run ios```, it works but always opening default device (iPhone SE in my case) and I'm not able to test app on other devices.

Am I doing something wrong? Is there any other way to target specific device?

**Comment by cpetrov** [comment]
Currently not, but it would be a very useful feature to have :+1: 

To achieve the same result for iOS, one would have to execute:
```sh
$ tabris build ios
$ xcrun instruments -w <device ID> # you can get the simulator ID by running "xcrun simctl list"
$ xcrun simctl install booted <path to the built .app file>
$ xcrun simctl launch booted <app ID>
```

It would be nice if we could reduce those to `tabris run ios --target=...` and provide a command to list all available devices. Those commands/flags would just delegate to their `cordova` equivalents.

**Comment by cpetrov** [comment]
This feature has been implemented and will be available in the upcoming version 2.2.0 of Tabris.js CLI.
